src/screens/Notifications/AddNotificationEmailScreen.tsx

```diff
@@ -46,8 +46,23 @@ class AddNotificationEmailScreen extends PureComponent<Props, State> {
     email: '',
   };
 
+  addListenerResetStateOnNavigationFocus = () =>
+    this.props.navigation.addListener('focus', () => {
+      this.setState({ email: '' });
+    });
+
+  removeListenerResetStateOnNavigationFocus = () =>
+    this.props.navigation.removeListener('focus', () => {
```

Comment:
it won't work, you have have reference for the same function,  something like:
``` 
focusHandler =() => {
      this.setState({ email: '' });
    });
    
this.props.navigation.addListener('focus', this.focusHandler);
    
this.props.navigation.removeListener('focus', this.focusHandler);

Reply:
up